models/intermediate/intermediate.yml

```diff
@@ -0,0 +1,34 @@
+version: 2
+
+models:
+    - name: int_microsoft_ads__ad_history
```

Comment:
It looks like `is_most_recent_version` is missing from the yml documentation. Would you be able to add that field and the description.